### PR TITLE
Update Vagrant Config

### PR DESCRIPTION
**Pull request by gtowey**
### Description

With a fresh install of Vagrant 2.0.1, and
VirtualBox 5.2.2r119230 on OSX the vagrant config
failed.  In particular the host machine was unable
to connect via ssh because the network to the VM
was unreachable.

I changed some of the extra config options passed
to VirtualBox for network config and it solved the issue.

In addition there were a some path issues -- it looks like
the build.sh script has changed and needs to be able to
run a `git revlist` to build the package.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -16,20 +16,21 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = BOX
   config.vm.box_download_insecure = true
   config.vm.box_check_update = false
-  config.vm.provider 'virtualbox' do |vb|
-    vb.customize ['modifyvm', :id, '--nictype1', 'virtio']
-    vb.customize ['modifyvm', :id, '--nictype2', 'virtio']
-  end
+#  config.vm.provider 'virtualbox' do |vb|
```
Comment (shlomi-noach) [inline Vagrantfile]: Should this code be eliminated? I'd like to avoid legacy commented code.

**Review by shlomi-noach — COMMENTED** [review]
Thank you! See question about commented code

**Comment by gtowey** [comment]
@shlomi-noach do you think this can be merged now?

**Comment by shlomi-noach** [comment]
Thank you!

**Pull request merged**